htdocs/mrp/mo_production.php

```diff
@@ -724,6 +724,9 @@
 			}
 			print '</td>';
 		}
+		if ($permissiontodelete) {
+			print '<td></td>';
```

Comment:
@Hystepik Avoid print '<td/>'
This is valid with XML but not with HTML5. Only tag that can be empty accept the short closing / syntax like img, br, ..
Even if most browsers understand, best is to never use it.